src/charm.py

```diff
@@ -146,7 +201,12 @@ def _on_collect_unit_status(self, event: CollectStatusEvent):  # noqa C901
             logger.info("The following configurations are not valid: %s", invalid_configs)
             return
 
-        for relation in ["fiveg_nrf", "database", "certificates"]:
+        for relation in [
+            FIVEG_NRF_RELATION_NAME,
+            "database",
```

Comment:
This could be a global variable, just like the other relations

Reply:
Yes, this could be also a Global variable, however  `database` relation is going to be removed by another open PR: https://github.com/canonical/sdcore-amf-k8s-operator/pull/223/files. That's reason, I did not change it.